### DISCUSSION TITLE
Few minor issues need some help!!

### DISCUSSION THREAD

**Discussion by maharajg** [prasanthrangan/hyprdots#183] [Q&A]
Hi,

This is working beautifully except for a few minor niggles on my laptop, 

1. Battery notification alerts not working, I've changed the battery name in batterynotify.sh and the command gives output on terminal.
2. How to dim screen, and turn off display after a few minutes? Googled and tried using this, but doesnt work.
 
`#!/bin/bash
swayidle -w \
timeout 120 'swaylock' \
timeout 400 'hyprctl dispatch dpms off' \
timeout 900 'systemctl suspend' \
resume 'hyprctl dispatch dpms on' \
before-sleep 'swaylock'`

Thanks and Great work.

**Comment by prasanthrangan** [comment]
> 1. Battery notification alerts not working, I've changed the battery name in batterynotify.sh and the command gives output on terminal.

any errors while running it as a daemon? you can try using `nohup` to capture the logs for daemon, just to test it.


> 2\. How to dim screen, and turn off display after a few minutes? Googled and tried using this, but doesnt work.

is swayidle working? test with monitor off/on as below -
`swayidle -w timeout 60 'hyprctl dispatch dpms off' resume 'hyprctl dispatch dpms on'`
